### DISCUSSION TITLE
bun build deletes "unused" code that is used by my HTML

### DISCUSSION THREAD

**Discussion by allquixotic** [oven-sh/bun#10953] [Q&A]
I have functions that are event handlers set up in my HTML code for a client-side JS I'm trying to bundle with `bun build`. It deletes all of my unused code when it does whatever it does, and I have `minify` set to `false`. That code is unused from the perspective of JS world, but it is absolutely used from the HTML side, and I need it. Why is it doing that and how do I stop it from doing it?

**Comment by VictorVSa** [comment]
I have the same problem...

**Reply by VictorVSa** [reply]
For the time being, I'm using something like this to make it think it's being used, and not trigger LSP errors

```typescript
const keep = (fn: Function) => fn;
keep(functionName);
```
